qsimcirq/qsim_circuit.py

```diff
@@ -105,7 +105,7 @@ def translate_cirq_to_qsim(
           qsim_gate = "rz"
           qsim_params = str(op.gate.exponent*np.pi)
         elif isinstance(op.gate, cirq.ops.CZPowGate) \
-                and op.gate.exponent == 1.0:
+                and ( op.gate.exponent == 1.0 or op.gate.exponent == -1.0 ):
```

Comment:
This is true for many of the gates listed here. Instead, I'd suggest doing something similar to the above gates by adding a general case (in addition to the existing `cz` case):
```
elif isinstance(op.gate, cirq.ops.CZPowGate):
  qsim_gate = "cp"
  qsim_params = str(op.gate.exponent*np.pi)
```
qsim uses `cp` to denote the controlled-phase gate - i.e. CZPowGate.

Reply:
Good point. Instead of using a special instance of CZPowGate, the next suggestion uses the cp gate in qsim instead.